src/libpgagroal/pipeline_session.c

```diff
@@ -220,6 +220,11 @@ session_client(struct ev_loop *loop, struct ev_io *watcher, int revents)
    {
       if (likely(msg->kind != 'X'))
       {
+         if (msg->kind == 'Q' || msg->kind == 'P' || msg->kind == 'E')
```

Comment:
`P` is Parse which we shouldn't count as a query, as `B` (Bind) won't need it after 5 statements.

Lets start with `Q` and `E` in this category.